src/main/java/org/testng/reporters/XMLStringBuffer.java

```diff
@@ -305,6 +305,9 @@ public void setDefaultComment(String defaultComment) {
    * Add a CDATA tag.
    */
   public void addCDATA(String content) {
+    if (content != null) {
+        content = content.replaceAll("\\p{Cc}", "");
```

Comment:
If possible, `\u000C` should be replaced by `&#xc;`:
http://www.fileformat.info/info/unicode/char/000c/index.htm

Reply:
I am basically using the regex pattern which stands for control characters and I believe `\u000C` is a control character (Form feed)

Reply:
Yes, but we are loosing information. Is it possible to do better?
If no, it is not a problem :)

Reply:
What sort of information are you referring to as being lost ? I didn't get that part. Can you please elaborate ?

Reply:
`"&#xc;"` and `""` are not the same thing.
But I agree the difference is not important and can be skipped.